src/Mesh.cpp

```diff
@@ -68,6 +68,12 @@ void Mesh::createLattice(int num_x, int num_y, int num_z) {
   _lattice->computeSizes();
 }
 
+
+/**
+ * @brief Set the _lattice of a mesh to be an existing one.
+ *        the user-input dimensions
```

Comment:
The user inputs dimensions
or 
existing one, for which the user inputs the dimensions

Reply:
changed